Patch by Robert Scheck <robert@fedoraproject.org> for libvmime >= 0.7.1, which adds a missing include file
in order to build successfully with GCC 4.4 and newer.

--- libvmime-0.7.1/src/platforms/posix/posixFile.cpp			2005-04-09 10:42:46.000000000 +0200
+++ libvmime-0.7.1/src/platforms/posix/posixFile.cpp.gcc-4.4-support	2009-03-29 21:09:07.000000000 +0200
@@ -28,6 +28,7 @@
 
 #include <dirent.h>
 
+#include <stdio.h>
 #include <string.h>
 
 #include "vmime/exception.hpp"
